app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -128,9 +144,9 @@ params:
 
 ### How it works
 
-When enabled, this plugin will monitor API traffic and log to
-[Moesif API Analytics](https://www.moesif.com/?language=kong-api-gateway). Use Moesif
-to understand API adoption and resolve issues quickly.
+When enabled, this plugin will capture API traffic and log to
+[Moesif API Analytics](https://www.moesif.com/?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong). 
+This plugin adds zero latency and does not sit in line to API traffic.
```

Comment:
what does "does not sit in line to API traffic" mean? It doesn't sit in a queue? It doesn't interfere with API traffic? Both?